release-1.8/release_notes_draft.md

```diff
@@ -271,18 +278,35 @@ kind.
 
 #### Storage
 
-* Capacity Isolation/Resource Management for Local Ephemeral Storage
-* Block Volumes Support
-* Enable containerization of mount dependencies
-* Support Attach/Detach for RWO volumes such as iSCSI, Fibre Channel and RBD
-* Volume Plugin Metrics
-* Snapshots
-* Resizing Volume Support
-* Exposing StorageClass Params To End Users (aka Provisioning configuration in PVC)
-* Mount Options to GA
-* Allow configuration of reclaim policy in StorageClass
-* Expose Storage Usage Metrics
-* PV spec refactoring for plugins that reference namespaced resources: Azure File, CephFS, iSCSI, Glusterfs
+* [stable] Mount Options
+  * The ability to speciy mount options has been promoted to stable.
+  * A new `MountOptions` field in the `PersistentVolume` spec can be used to specify mount options (instead of an annotation).
+* [stable] Support Attach/Detach for RWO volumes iSCSI and Fibre Channel.
+* [stable] Expose Storage Usage Metrics
+  * Expose how much available capacity a given PV has through the Kubernetes metrics API.
+* [stable] Volume Plugin Metrics
+  * Expose success and latency metrics for all Kubernetes mount/unmount/attach/detach/provision/delete calls through the Kubernetes metrics API.
+* [stable] Modify PV spec for Azure File, CephFS, iSCSI, Glusterfs to allow referencing namespaced resources.
+* [beta] Reclaim policy in StorageClass
+  * Allow configuration of reclaim policy in StorageClass, instead of always defaulting to `delete` for dynamically provisioned volumes.
+* [alpha] Resizing of Volumes
+  * Enable increasing the size of Volumes through the Kubernetes API.
+  * For alpha, this feature only increases the size of the underlying volume and does not do filesystem resizing.
```

Comment:
Out of curiosity, shouldn't it be transparent to kube, and up to the backend storage implementation what they do with a volume resize / expand request? (In the sense of: the backend storage implementation may just do the filesystem resizing for you if required, but it may not even have a classical filesystem underneath at all and may do something entirely different...) I.e. the notion of filesystem and filesystem resize seems misplaced at the kube-layer.

Reply:
Unfortunately, for most volume providers resizing is a 2 step process. One that happens controller-manager's control plan and another that involves file system resizing on kubelet. https://github.com/kubernetes/community/pull/657 may be right place to discuss it

Reply:
@gnufied thanks!

Reply:
Kubernetes exposes volumes as file or block. For filesystem volumes, in order for an end user to actually make use of the newly available extra space, a filesystem resize is required. If volume resizing is to be part of the k8s storage API, then we need to make sure we automate this as well.